client/icecc-create-env.in

```diff
@@ -27,6 +27,29 @@ is_contained ()
   esac
 }
 
+# Equivalent to Linux readlink -f command
+resolve_path ()
+{
+    TARGET_FILE=$1
+
+    cd `dirname $TARGET_FILE`
```

Comment:
use pushd here and popd at the end of the function.  Or run this all in a subshell.